### DISCUSSION TITLE
how to disable clipboard sharing from command line or config file?

### DISCUSSION THREAD

**Discussion by joaociocca** [debauchee/barrier#2083] [Q&A]
I may just be dumb, but I can't seem to find documentation about this... is there a way, thru command-line option of config file, to disable the clipboard sharing? I know that using the GUI we can uncheck the respective option, but this seems to be enabled by default on command line and it's interfering with other clipboard tools.

**Comment by DeepYogurt** [comment]
On a Mac running as the Barrier server, click on the "Configure Server" button - then click the "Advanced Server Settings" and in there is a checkbox for enabling 'Clipboard Sharing" along with several other settings. here is what the dialog box looks like for me:

![image](https://github.com/user-attachments/assets/16e189a8-3fa5-47b7-b6db-f1cd4cb630c7)

Hope that helps...

-bob

**Reply by DeepYogurt** [reply]
Forgot to mention that this was on Barrier 2.4.0 on macOS Mojave

**Reply by joaociocca** [reply]
thanks, but I'd be running in CLI/headless/no GUI mode both server and client, and both on Linux...

**Comment by DeepYogurt** [comment]
Sorry, I missed your mention of wanting a way without the GUI - in my Barrier config file, in the section labeled "Options" have the following:

section: options
        relativeMouseMoves = false
        screenSaverSync = true
        win32KeepForeground = false
        clipboardSharing = true
        switchCorners = none +top-left +top-right +bottom-left +bottom-right
        switchCornerSize = 10
        keystroke(Alt+m) = ;switchToScreen(mini.local)
        keystroke(Alt+i) = ;switchToScreen(iMac.local)
        keystroke(Alt+t) = ;switchToScreen(TCMacPro.local)
end

So am guessing that the "clipboardSharing = false" statement is what you might need...

Good luck...

-bob
